XTS/WSCF/classes/com/arjuna/mwlabs/wscf/model/sagas/arjunacore/BACoordinator.java

```diff
@@ -428,7 +434,7 @@ public CoordinatorId identifier () throws SystemException
 
     public String type ()
     {
-        return "/StateManager/BasicAction/AtomicAction/Sagas/BACoordinator";
+        return TYPE;
```

Comment:
This is normally made so different implementations can override the type() method to go somewhere else in the objectstore - however you have a getType() on BACoordinator so it means no one could ever override the getType() from what I can see, maybe I miss something or maybe its not required but I do need some clarification